### ISSUE TITLE
Incompatibility with ruby-kafa 0.7.6

### ISSUE THREAD

**Issue by vixns** [fluent/fluent-plugin-kafka#247]
since https://github.com/zendesk/ruby-kafka/commit/ffb6c50e6fa4ca4877858eb3edb88fe81eceb06c

```
[...]
<source>
  @type kafka_group
  brokers test:9092
  topics test
  format text
  consumer_group test
</source>
[...]
```

```
2019-03-17 11:59:04 +0000 [error]: #0 unexpected error error_class=NoMethodError error="undefined method `formatter' for #<Fluent::Log:0x007f1379faac38>\nDid you mean?  format\n               format_time\n               format="
  2019-03-17 11:59:04 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/ruby-kafka-0.7.6/lib/kafka/tagged_logger.rb:57:in `new'
  2019-03-17 11:59:04 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/ruby-kafka-0.7.6/lib/kafka/client.rb:75:in `initialize'
  2019-03-17 11:59:04 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/ruby-kafka-0.7.6/lib/kafka.rb:366:in `new'
  2019-03-17 11:59:04 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/ruby-kafka-0.7.6/lib/kafka.rb:366:in `new'
  2019-03-17 11:59:04 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluent-plugin-kafka-0.9.0/lib/fluent/plugin/in_kafka_group.rb:160:in `start'
  2019-03-17 11:59:04 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.4.0/lib/fluent/compat/call_super_mixin.rb:42:in `start'
  2019-03-17 11:59:04 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.4.0/lib/fluent/root_agent.rb:203:in `block in start'
  2019-03-17 11:59:04 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.4.0/lib/fluent/root_agent.rb:192:in `block (2 levels) in lifecycle'
  2019-03-17 11:59:04 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.4.0/lib/fluent/root_agent.rb:191:in `each'
  2019-03-17 11:59:04 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.4.0/lib/fluent/root_agent.rb:191:in `block in lifecycle'
  2019-03-17 11:59:04 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.4.0/lib/fluent/root_agent.rb:178:in `each'
  2019-03-17 11:59:04 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.4.0/lib/fluent/root_agent.rb:178:in `lifecycle'
  2019-03-17 11:59:04 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.4.0/lib/fluent/root_agent.rb:202:in `start'
  2019-03-17 11:59:04 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.4.0/lib/fluent/engine.rb:274:in `start'
  2019-03-17 11:59:04 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.4.0/lib/fluent/engine.rb:219:in `run'
  2019-03-17 11:59:04 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.4.0/lib/fluent/supervisor.rb:799:in `run_engine'
  2019-03-17 11:59:04 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.4.0/lib/fluent/supervisor.rb:549:in `block in run_worker'
  2019-03-17 11:59:04 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.4.0/lib/fluent/supervisor.rb:724:in `main_process'
  2019-03-17 11:59:04 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.4.0/lib/fluent/supervisor.rb:544:in `run_worker'
  2019-03-17 11:59:04 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.4.0/lib/fluent/command/fluentd.rb:316:in `<top (required)>'
  2019-03-17 11:59:04 +0000 [error]: #0 /usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
  2019-03-17 11:59:04 +0000 [error]: #0 /usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
  2019-03-17 11:59:04 +0000 [error]: #0 /var/lib/gems/2.3.0/gems/fluentd-1.4.0/bin/fluentd:8:in `<top (required)>'
  2019-03-17 11:59:04 +0000 [error]: #0 /usr/local/bin/fluentd:22:in `load'
  2019-03-17 11:59:04 +0000 [error]: #0 /usr/local/bin/fluentd:22:in `<main>'
```

**Comment by repeatedly** [comment]
Thanks for the report. Will fix soon.

**Comment by repeatedly** [comment]
Released v0.9.1 with this fix.
